### DISCUSSION TITLE
Formalise approach to tackle new responsibilities and to set KPIs

### DISCUSSION THREAD

**Discussion by maoo** [finos/technical-oversight-committee#79] [General]
During the [last TOC call](https://github.com/finos/technical-oversight-committee/issues/76) the group have identified the need to formalise an approach to tackle new responsibilities, out of the list defined on https://github.com/finos/technical-oversight-committee/blob/master/responsibilities.md

The list of quarterly recurring items is the following:
- Solicit and recommend cross-project synergies
- Review of project health
- Review of project alignment with FINOS strategy
- Identify use-cases / areas of interest (to bring in front of Board of Directors)
- Provide strategic input based on existing landscape components

For each of these items, we need to better understand:
- What is expected as outcome
- What are the activities that are expected by TOC members
- How to measure activity and progress
- Goals and milestones

This discussion intends to gather feedback from the TOC group.

**Comment by yassinelaouni** [comment]


**Reply by yss-fgggg** [reply]
you are good
